docs/src/main/paradox/docs/delta/api/practice.md

```diff
@@ -0,0 +1,51 @@
+# Practice
```

Comment:
Is it worth linking to the validation endpoint? Or it's better not to mix them here yet?